cunumeric/random/generator.py

```diff
@@ -28,6 +28,29 @@
 
 class Generator:
     def __init__(self, bit_generator: BitGenerator) -> None:
+        """
+        Generator(bit_generator)
+
+        Container for the BitGenerators.
+
```

Comment:
I would also transfer this line from the NumPy docs:
```
:class:`cunumeric.random.Generator` exposes a number of methods for generating random numbers drawn from a variety of probability distributions.
```
IMHO just saying "Container for the BitGenerators" doesn't describe well what this class does.

Reply:
3f68ce4aacfbb28323d4ef611e7badaccfed9ac4